shared/templates/grub2_bootloader_argument_absent/oval.template

```diff
@@ -32,6 +32,7 @@
 <def-group>
   <definition class="compliance" id="{{{ _RULE_ID }}}" version="2">
     {{{ oval_metadata("Ensure " + ARG_NAME + " is not set in the kernel line in /etc/default/grub.") }}}
+    <criteria operator="OR">
```

Comment:
shouldn't this be "AND"?

I'm thinking that if the system supports bootable containers and it doesn't contain the argument in `/usr/lib/bootc/kargs.d/*.toml` it will pass even if the argument is present somewhere else

Reply:
Good point! I will try to adjust this OVAL to prevent that situation.